test/js_test.go

```diff
@@ -2909,18 +2917,6 @@ func testJetStream_ClusterReconnectDurableQueueSubscriber(t *testing.T, subject
 				// Do not ack and wait for redelivery on reconnect.
 				srvA.Shutdown()
 				return
-			case count == 11:
-				// Do another Shutdown of the server we are connected with.
-				switch nc.ConnectedUrl() {
-				case srvAClientURL:
-					srvA.Shutdown()
```

Comment:
This second shutdown was making the test too unstable so decided to remove it for now.